platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -300,13 +284,6 @@ private static synchronized Executor getCleaningPool(@NonNull final VirtualMapCo
      */
     private volatile ConcurrentArray<Mutation<Long, K>> dirtyLeafPaths = new ConcurrentArray<>();
```

Comment:
Volatile fields were not reverted, but I think it's not important

Reply:
Strictly speaking, old behavior (no volatile) is a bug. The field is accessed from multiple threads, so it should be either final, or volatile, or there should be some synchronization. So I don't think this change needs a revert